polus-precompute-slide-plugin/src/utils.py

```diff
@@ -276,22 +515,57 @@ def bfio_metadata_to_slide_info(bfio_reader,outPath):
     }
     
     # initialize the json dictionary
-    info = {
-        "data_type": dtype,
-        "num_channels":1,
-        "scales": [scales],       # Will build scales below
-        "type": "image"
-    }
-    
+    if imagetype == "segmentation":
+        info = {
+            "data_type": dtype,
+            "num_channels":1,
+            "scales": [scales],       # Will build scales below
+            "type": imagetype,
+            "segment_properties": "infodir"
+        }
+    else:
+        info = {
+            "data_type": dtype,
+            "num_channels":1,
+            "scales": [scales],       # Will build scales below
+            "type": imagetype,
+        }
```

Comment:
This can be simplified to:
```python
info = {
    "data_type": dtype,
    "num_channels": 1,
    "scales": [scales],
    "type": imagetype,
}
if imagetype == "segmentation":
    info["segment_properties"] = "infodir"
```

I know this seems trivial, but it makes the code easier to read and change in the future. For example, if in the future you need to change something like `num_channels`, you will have to change it in two places with your code or in one place with the recommended revision I suggested here.

There is a general philosophy that code is read by humans more than it is written. With this in mind, we should approach writing our code with the same mindset as writing a paper. The easier we make it to understand by removing redundant or runon sentences, the easier we will make it on our future selves.